mage/query-modules/python/igraphalg.md

```diff
@@ -0,0 +1,258 @@
+---
+id: igraphalg
+title: igraphalg
+sidebar_label: igraphalg
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+export const Highlight = ({children, color}) => (
+  <span
+style={{
+  backgroundColor: color,
+  borderRadius: '2px',
+  color: '#fff',
+  padding: '0.2rem',
+}}>
+{children}
+  </span>
+);
+
+[![docs-source](igraphalg)](https://github.com/memgraph/mage/blob/main/python/igraphalg.py)
+
+## Abstract
+
+This module, named **igraphalg**, provides a comprehensive set of thin wrappers around some of the algorithms present in the [Igraph](https://igraph.org/) package. The wrapper functions now have the capability to create a Igraph compatible graph-like object that can stream the native database graph directly saving on memory usage significantly.
```

Comment:
Nice to see igraph being used in more and more projects! :-) Just wanted to point out that igraph prefers the lowercase spelling.